### DISCUSSION TITLE
如何在右键菜单添加一个“加入播放列表”的选项

### DISCUSSION THREAD

**Discussion by KangMz** [hooke007/mpv_PlayKit#398] [mpv 通用]
就类似其他播放器一样可以选择文件添加进播放列表。目前的安装的install.bat中，只有一个播放的选项，如果选中多个然后点击播放的话，会同时打开多个窗口。。这不是我所期望的

**Comment by hooke007** [comment]
我不熟悉ps的接口，建议向上游提FR https://github.com/shinchiro/mpv-packaging

**Comment by izen000** [comment]
可以了解下MPV.NET

**Comment by KangMz** [comment]
好的，感谢！我去看看其他项目有没有相关的配置说明
